src/Resources/views/default/show_vertical.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends '@BaseEasyAdmin/default/show.html.twig' %}
```

Comment:
I was about to do it but:
https://github.com/EasyCorp/EasyAdminBundle/blob/master/src/Resources/views/default/show.html.twig
I need to remove div line 20:
<div class = "form-horizontal">
That's why {% block main%} and if I rewrite it would not be the changes to see EasyAdminExtension.

It is understood? I'm right?

Reply:
Hi @gonzakpo ,

Can you try that content for show_vertical.html.twig :

```
{% extends '@EasyAdminExtension/default/show.html.twig' %}

{% block main %}
    <div class="row">
        {% block show_fields %}
            {% for field, metadata in fields %}
                {% block show_field %}
                    <div class="form-group field-{{ metadata.type|default('default')|lower }} {{ metadata.css_class|default('') }}">
                        <label class="control-label">
                            {{ (metadata.label ?: field|humanize)|trans(_trans_parameters)|raw }}
                        </label>
                        <div class="form-control">
                            {{ easyadmin_render_field_for_show_view(_entity_config.name, entity, metadata) }}
                        </div>

                        {% if metadata.help|default('') != '' %}
                            <span class="help-block"><i class="fa fa-info-circle"></i> {{ metadata.help|trans|raw }}</span>
                        {% endif %}
                    </div>
                {% endblock %}
            {% endfor %}
        {% endblock %}

        <div class="form-group form-actions">
            <div class="col-sm-10 col-sm-offset-2">
            {# Do not display list action items if not granted #}
            {{ block('item_actions') }}
            </div>
        </div>
    </div>

    {{ block('delete_form') }}
{% endblock %}
```
If it's ok but you don't see why, I'll explain you.

Reply:
It works. I thought I was going to overwrite the block